src/commondao/common.dao.model.ts

```diff
@@ -208,6 +223,11 @@ export interface CommonDaoStreamForEachOptions<IN>
     TransformLogProgressOptions<IN> {}
 
 export interface CommonDaoStreamOptions extends CommonDaoOptions {
+  /**
+   * @default false (for streams). Setting to true enables deletion of immutable objects
+   */
+  overrideImmutability?: boolean
```

Comment:
`CommonDaoStreamOptions` already extends `CommonDaoOptions`where you already put this property, right? Then it's not needed here